src/nbl/asset/utils/CHLSLCompiler.cpp

```diff
@@ -208,9 +280,9 @@ core::smart_refctd_ptr<ICPUShader> CHLSLCompiler::compileToSPIRV(const char* cod
         hlslOptions.preprocessorOptions.logger.log("code is nullptr", system::ILogger::ELL_ERROR);
         return nullptr;
     }
-
+    std::vector<std::string> dxc_compile_flags = {};
```

Comment:
you could just initialize them to `hlslOptions.dxcFlags` thing or whatever that span member is called

Reply:
I was told to have any arguments read from `#pragma` override any arguments passed by command line, thus it has to be a new vector, that replaces hlslOptions.dxcOptions.